### PR TITLE
[IMP] Remove alias in JS files

### PR DESCRIPTION
**Pull request by bastien-ipb**
Description of the issue/feature this PR addresses:
 Remove all the aliases in the odoo-module of JS files and fix all the imports 
Current behavior before PR:
 
Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/portal/static/src/js/portal_signature.js [hunk 1]
```diff
@@ -189,6 +189,4 @@ publicWidget.registry.SignatureForm = publicWidget.Widget.extend({
     },
 });
 
-export default {
-    SignatureForm: SignatureForm,
-};
+export default SignatureForm
```
Comment (ged-odoo) [inline addons/portal/static/src/js/portal_signature.js:192]: ;

**Comment by william-andre** [comment]
Hey, just out of curiosity and sorry for intruding :see_no_evil: 
I'm a bit of a JS-noob and I'm just wondering why this change is made

File: debian/odoo.conf [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [options]
 ; This is the password that allows database operations:
-; admin_passwd = admin
+admin_passwd = admin
```
Comment (ged-odoo) [inline debian/odoo.conf:3]: erm, remove this

**Comment by ged-odoo** [comment]
> Hey, just out of curiosity and sorry for intruding see_no_evil I'm a bit of a JS-noob and I'm just wondering why this change is made

to move the odoo ecosystem closer to the standard javascript ecosystem. With normal imports, tooling/IDE can autocomplete symbols. It also make it easier to map the name of the module to the corresponding file (for example, `@web/core/utils/strings` is in `addons/web/static/src/core/utils/strings`.

**Comment by ged-odoo** [comment]
@odoo/rd-security There are 3 checks to validate:
- `self._getCommentContainer($source).html($(qweb.render("portal_rating.chatter_rating_publisher_comment", {` => the change is just an extra white space that has been removed
- `const _Markup = owl.markup('').constructor;` => same
- `export function Markup(v, ...exprs) {` => the function is just now directly exported (in addition to the default export)

It seems to me that these changes are obviously not a security risk, so I allow myself to clear the check. If you disagree, don't hesitate to act accordingly.
@robodoo override=ci/security

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by ged-odoo** [comment]
robodoo r+

**Comment by rdeodoo** [comment]
@ged-odoo Maybe the commit message could be properly formatted?
image

Maybe even include the PR description inside it `Remove all the aliases in the odoo-module of JS files and fix all the imports`
There might also be a task that was forgotten to be added?

**Review by AntoineVDV — APPROVED** [review]

**Comment by bastien-ipb** [comment]
robodoo r+

**Comment by bastien-ipb** [comment]
robodoo r+

**Comment by seb-odoo** [comment]
@robodoo p=1
Better avoid further conflicts!

**Comment by Xavier-Do** [comment]
@robodoo p=2

**Comment by Xavier-Do** [comment]
Can you please keep this for after the freeze? If you didn't had the conflict, another pr would have, and right now we are waiting for the last pr that should go in the freeze.

**Comment by bastien-ipb** [comment]
robodoo r+

**Pull request closed without merging**